CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to the APM Agent
+
+The APM Agent is open source and we love to receive contributions from our community — you!
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code.
+
+You can get in touch with us through [Discuss](https://discuss.elastic.co/c/apm), feedback and ideas are always welcome.
+
+## Code contributions
+
+If you have a bugfix or new feature that you would like to contribute, please find or open an issue about it first.
+Talk about what you would like to do.
+It may be that somebody is already working on it, or that there are particular issues that you should know about before
+implementing the change.
+
+You will have to fork the `apm-agent-nodejs` repo, please follow the instructions in the [readme](README.md).
```

Comment:
We don't currently have instructions in the README for this... not sure if it would make more sense to add that info directly here or if we should have a "Local development" section or something similar in the README 🤔

Reply:
I think it would make sense to move all development/testing related documentation to this file and then instead link to that from the README. For now just remove this line.